pkg/parca/parca.go

```diff
@@ -92,27 +93,16 @@ type Flags struct {
 
 	EnablePersistence bool `default:"false" help:"Turn on persistent storage for the metastore and profile storage."`
 
-	StorageGranuleSize  int64  `default:"26265625" help:"Granule size in bytes for storage."`
-	StorageActiveMemory int64  `default:"536870912" help:"Amount of memory to use for active storage. Defaults to 512MB."`
-	StoragePath         string `default:"data" help:"Path to storage directory."`
-	StorageEnableWAL    bool   `default:"false" help:"Enables write ahead log for profile storage."`
-	StorageRowGroupSize int    `default:"8192" help:"Number of rows in each row group during compaction and persistence. Setting to <= 0 results in a single row group per file."`
+	Storage FlagsStorage `embed:"" prefix:"storage-"`
+
+	Symbolizer FlagsSymbolizer `embed:"" prefix:"symbolizer-"`
 
-	SymbolizerDemangleMode  string `default:"simple" help:"Mode to demangle C++ symbols. Default mode is simplified: no parameters, no templates, no return type" enum:"simple,full,none,templates"`
-	SymbolizerNumberOfTries int    `default:"3" help:"Number of tries to attempt to symbolize an unsybolized location"`
+	DebugInfod FlagsDebugInfod `embed:"" prefix:"debug-infod-"`
```

Comment:
```suggestion
	DebugInfod FlagsDebugInfod `embed:"" prefix:"debuginfod-"`
```

We agreed to use `debuginfo` as a whole word (https://github.com/parca-dev/parca/pull/2147).
This also changes the existing flag name so let's stick to the `debuginfo`

Reply:
Nice, I like that. Perfect.